whylog_vim/tests/tests_output_formater/tests_utils.py

```diff
@@ -0,0 +1,6 @@
+from whylog_vim.output_formater.utils import to_buttons
+
+
+def tests_to_buttons():
+    test_list = ['jajko', 'kanapka', 'ziemniak', 'mleko']
```

Comment:
Because maybe one day English speaking guys will read this code and they can be confused